test/steps/files/1_given.js

```diff
@@ -177,7 +177,18 @@ export function theFileIsNotValidJSON() {
 	fsUtils.readJSONSync.throws('Invalid JSON');
 }
 
-export function theFileIsValidJSON() {
+export function theFileIsMissingRequiredKeys() {
+	const userConfig = {
+		name: 'custom-name',
```

Comment:
maybe check the nested values too?

Reply:
ah thats true, as long as the key exists, it should be able to set.
I think it's good then